javascript-loops-practice/filter-out-strings.js

```diff
@@ -0,0 +1,11 @@
+/* exported filterOutStrings */
+
+function filterOutStrings(values) {
+  var numbersOnlyArray = [];
+  for (let i = 0; i < values.length; i++) {
+    if (typeof values[i] === 'number') {
```

Comment:
This function isn't supposed to collect all numbers, it's supposed to collect all values that are not strings.